### DISCUSSION TITLE
2021 model 3 with power trunk unable to close trunk with /command/actuate_trunk endpoint

### DISCUSSION THREAD

**Discussion by samleemc** [timdorr/tesla-api#418] [Q&A]
Hi there,

I have a 2021 model 3 with power trunk.
According to the API doc, I tried to call the below endpoint to try to close the trunk. But the API is giving me the below error.

https://owner-api.teslamotors.com/api/1/vehicles/{id}/command/actuate_trunk

```
{
  "reason": "unknown error",
  "result": false
}
```

Anyone know another endpoint to open/close trunk of the new model 3 please?

Thanks a lot!

**Comment by samleemc** [comment]
Hi, Anyone with any knowledge to share please?

**Reply by dacastro4** [reply]
Does it open?

**Reply by dacastro4** [reply]
Did you send `which_trunk`?

**Reply by samleemc** [reply]
Yes. I do. The error response is 
{"error":"invalid_command","response":null,"error_description":""}
![44251CA0-DC6D-47AB-ACAA-3F54A65D16FE](https://user-images.githubusercontent.com/42374700/128518213-c34ad9a3-4ca0-403d-ae04-720344866950.jpeg)
![9F856DA9-2546-44BF-A79D-1B392D3B6AAC](https://user-images.githubusercontent.com/42374700/128518221-df1091dc-cace-4787-9c94-3e86c937a881.jpeg)

**Reply by dacastro4** [reply]
It has always worked with that user agent? Did you try removing it?

**Reply by samleemc** [reply]
It works when call GET /api/1/vehicles/{is}/data_request/vehicle_state

and I believe the API document said it require a “User-Agent” for all the API call?

**Comment by sweetdream-001** [comment]
I can open frunk using actuate_trunk endpoint but I can't close it.
I have Tesla Model S and CyberTrunk.
How do I solve this?
Where can I go to see reviews or opinions about Tesla cars?
